src/hooks/use-funnel.tsx

```
@@ -98,14 +98,16 @@ const useFunnel = <K extends Record<string, unknown>, T extends Extract<keyof K,
   };
 
   const currentStep = getInitialStep(stepInQueryParam, initialStep);
-  const [funnelData, setFunnelData] = useState<Partial<K[T]>>({});
+
+  const funnelDataRef = useRef<Partial<K[T]>>({});
+  let funnelData = funnelDataRef.current;
```

ESLint

[error] 'funnelData' is never reassigned. Use 'const' instead.

(prefer-const)